packages/flutter/lib/src/widgets/mouse_detector.dart

```diff
@@ -0,0 +1,176 @@
+///  Copyright 2018 The Chromium Authors. All rights reserved.
+///  Use of this source code is governed by a BSD-style license that can be
+///  found in the LICENSE file.
+
+import 'package:flutter/foundation.dart';
+import 'package:flutter/gestures.dart';
+import 'package:flutter/rendering.dart';
+
+import 'annotated_region.dart';
+import 'framework.dart';
+
+/// A widget that detects changes in mouse pointer positions when a mouse button
+/// is not pressed. Note that this only applies to mouse pointers: touch
```

Comment:
nit: strike "note that this"; see https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#avoid-empty-prose

Reply:
Note that saying "note that" is a habit I'm trying to break. :-)